frontend/templates/ng-templates/browser-region.html

```diff
@@ -8,19 +8,20 @@
     <div class="col-md-12">
         <h1 ng-if="ctrl.rsid">{{ ctrl.rsid }}</h1>
         <h1 ng-if="!ctrl.rsid">Region: {{ ctrl.region.chrom }}
+	    <span ng-if="ctrl.region.statusCode">{{ ctrl.region.variantId }}</span>
             <span ng-if="ctrl.region.stop"> / {{ ctrl.region.start }} / {{ ctrl.region.stop }}</span>
         </h1>
     </div>
 
     <!-- WARNINGS -->
-    <div ng-if="ctrl.region.stop">
-        <div ng-if="ctrl.region.stop - ctrl.region.start > 100000">
-            <div class="alert alert-danger col-md-offset-3 col-md-6">
-                <h3>Region too large!</h3>
-                <p>Your region is too large. Please submit a region of at most 100 kb.</p>
-                <p>If you require larger regions, please see our <a href="/dataset/{{ ctrl.dataset.shortName }}/download">Download</a> page for raw data.</p>
-            </div>
+    <div ng-if='ctrl.region.statusText=="The region is too large"'>
```

Comment:
I'm not sure about comparing with a string in this way here. Isn't there a statusCode for too large region?

Reply:
I'm using 400 (`bad request`) for too large region. 400 is also used for problems with parsing the region (which in my opinion also fits `bad request`). Thus the need to check the actual text. But maybe I should change it to `===`?